File: src/main/kotlin/moe/kurenai/tdlight/model/media/ChatMemberMember.java
package moe.kurenai.tdlight.model.media;

import com.fasterxml.jackson.annotation.JsonProperty;
import moe.kurenai.tdlight.model.chat.ChatMember;
import moe.kurenai.tdlight.model.message.User;

public record ChatMemberMember(
        @JsonProperty("status") String status,
        @JsonProperty("user") User user
) implements ChatMember {
}

File: src/main/kotlin/moe/kurenai/tdlight/model/media/ChatMemberMember.kt
package moe.kurenai.tdlight.model.media

import com.fasterxml.jackson.annotation.JsonProperty
import moe.kurenai.tdlight.model.chat.ChatMember
import moe.kurenai.tdlight.model.message.User

class ChatMemberMember : ChatMember() {
    @JsonProperty("status")
    var status: String = "member"

    @JsonProperty("user")
    var user: User? = null
}